operator/roles/forkliftcontroller/templates/monitor/recordingrole-migrations.yml.j2

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ metrics_rule_name }}
+  namespace: {{ app_namespace }}
+spec:
+  groups:
+  - name: mtv-migrations
+    rules:
+    - record: cluster:mtv_migrations_status_total:max
+      expr: max by(status, provider, mode, target) (mtv_migrations_status_total)
```

Comment:
are we sure we need all 4 labels? telemetry team is very strict about the number of records added per query. if we can aggregate by less it will be better.

Reply:
These labels are needed. It should be fine.